### PR TITLE
fix: 修复 form-data 依赖的缺失；增加 cross-env 以兼容 Windows 和 Linux 的环境变量设置

### PR DESCRIPTION
**Pull request by CaoMeiYouRen**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
NOROUTE
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [x] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

添加了 `form-data` 包，因为在 `lib\routes\telecompaper\news.ts` 文件中使用到了

添加了 `cross-env` 包。目前的环境变量设置方式不支持 Windows，故使用  `cross-env` 兼容 Windows 和 Linux 的环境变量设置

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
         "cheerio": "1.0.0-rc.12",
         "chrono-node": "2.7.5",
         "city-timezones": "1.2.1",
+        "cross-env": "^7.0.3",
```
Comment (DIYgod) [inline package.json:66]: Please update the `pnpm-lock.yaml` file and use the locked version.

```
"cross-env": "7.0.3",
```

**Review by DIYgod — CHANGES_REQUESTED** [review]

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+package-lock=true
```
Comment (TonyRL) [inline .npmrc:1]: The value of `package-lock` is true by default.
https://docs.npmjs.com/cli/v10/using-npm/config#package-lock

Comment (CaoMeiYouRen) [reply .npmrc:1]: > The value of `package-lock` is true by default. https://docs.npmjs.com/cli/v10/using-npm/config#package-lock

If the user overrides the default value of package-lock in the global configuration, the lock file will not be generated properly. 

Therefore, my recommendation is to set the default value of package-lock in the project

**Review by TonyRL — COMMENTED** [review]
CI is still failing. Please remove `pnpm-lock.yaml` and regenerate it using `pnpm i`

**Pull request merged**